src/libm/CMakeLists.txt

```diff
@@ -645,7 +645,11 @@ if(BUILD_INLINE_HEADERS)
       DEPENDS
       ${INLINE_HEADER_FILES_GENERATED}
       )
-    install(FILES ${INLINE_HEADER_FILES_GENERATED} DESTINATION include)
+    install(
+        FILES ${INLINE_HEADER_FILES_GENERATED}
+        DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
```

Comment:
Which targets do these headers belong to? Are they already included by one of the `INCLUDES DESTINATION`s?

Reply:
These files are supposed to be installed in the directory in which sleef.h is installed.

Reply:
Yes, but logically which target do they belong to?

Reply:
I think it is better to make a new target for these inline headers.

Reply:
Ah, I failed to notice the custom target right above. `add_custom_target` however creates a `UTILITY` target, which can't be installed. I'll have to convert it to an `INTERFACE` library target.

Reply:
I hit a snag, it seems the cuda header is not generated for me and because the header is missing, the install errors out.

Reply:
I don't understand your problem.
Please provide a sequence of commands you executed.

Reply:
I fixed the issue, it was just a missing build time dependency link.